cms/djangoapps/contentstore/utils.py

```diff
@@ -408,7 +410,7 @@ def get_user_partition_info(xblock, schemes=None, course=None):
             # Put together the entire partition dictionary
             partitions.append({
                 "id": p.id,
-                "name": p.name,
+                "name": unicode(p.name),  # Convert into a string in case ugettext_lazy was used
```

Comment:
Note to reviewers: I had to add this because the EnrollmentTrackUserPartition uses ugettext_lazy (due to the fact that it lives in common.lib.xmodule.xmodule). When CMS unit tests ran, p.name was a function instead of a string, and JSON encoding failed.